### PR TITLE
批量导入导出配置文件功能 for Swift 3

### PR DESCRIPTION
**Pull request by qinyuhang**
已解决冲突问题，请Merge

### CODE REVIEW

**Comment by qiuyuzhou** [comment]
Ok, I would check it later.

**Comment by qiuyuzhou** [comment]
Merge 后，发现菜单中“登录时自动启动”以及新增的3个导入导出菜单项都是disable状态

**Comment by qinyuhang** [comment]
都是disable？额？不会是， storyboard和function没有link起来？
我再push一个commit过来

**Comment by DeronLee** [comment]
一定要做好本地测试啊哈哈

File: ShadowsocksX-NG/ServerProfileManager.swift [hunk 1]
```diff
@@ -70,4 +70,114 @@ class ServerProfileManager: NSObject {
...
+                }
+            }
+        }
+    }
+    
+    func exportConfigFile() {
+        //读取example文件，删掉configs里面的配置，再用NSDictionary填充到configs里面
+        let fileManager = FileManager.default
+        
+        let filePath:String = Bundle.main.bundlePath + "/Contents/Resources/example-gui-config.json"
```
Comment (qiuyuzhou) [inline ShadowsocksX-NG/ServerProfileManager.swift]: Should get resource file path like this:
```
let src = Bundle.main.path(forResource: "gfwlist", ofType: "txt")
```

File: ShadowsocksX-NG/example-gui-config.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (qinyuhang) [inline ShadowsocksX-NG/example-gui-config.json]: this file is add to the project here, should check if the file is not copy to resource bundle in build

Comment (qinyuhang) [inline ShadowsocksX-NG/example-gui-config.json]: It is not added to the Xcode Project file

Comment (qiuyuzhou) [inline ShadowsocksX-NG/example-gui-config.json]: Because you haven't added it to project, so it not added to 'Copy Bundle Resources' build phase.

**Pull request merged**